sklearn/metrics/tests/test_common.py

```diff
@@ -638,7 +638,8 @@ def test_inf_nan_input():
     for metric in CLASSIFICATION_METRICS.values():
         for y_true, y_score in invalids:
             assert_raise_message(ValueError,
-                                 "Can't handle mix of binary and continuous",
```

Comment:
I'm definitely willing to change the test so that both the old and new message are tested. But I think that would defeat the purpose of this PR, don't you think?